salt/modules/mod_random.py

```diff
@@ -96,7 +96,7 @@ def str_encode(value, encoder="base64"):
     return out
 
 
-def get_str(length=20):
+def get_str(length=20, **kwargs):
```

Comment:
Please be explicit on the keyword arguments and their defaults. We don't really need this to be `**kwargs`